src/RA_Defs.cpp

```diff
@@ -62,6 +62,52 @@ std::string Narrow(const std::string& str)
     return str;
 }
 
+
+NativeString NativeStr(NonNativeCString str) noexcept
+{
+#ifdef UNICODE
+	return Widen(str);
+#else
+	return Narrow(str);
+#endif // UNICODE
+}
```

Comment:
Do you really need non-const versions of these functions?

Reply:
`NonNativeCString `and `NativeCString `are always `const`, if you mean `const `in the beginning there's no point.

Reply:
Nevermind. One takes a `NonNativeString` and the other is a `NonNativeCString`. (missed the C).